source/confirm_enrolment.html.md.erb

```diff
@@ -0,0 +1,31 @@
+---
+title: Enrol Candidate
+weight: 50
+---
+
+# Confirm candidate enrolment
+Used to confirm the candidate enrolment.
+
+```
+POST /applications/:uuid/confirm_enrolment
+```
+
+## Example request
+
+```
+POST /applications/aa546741-b54a-403c-a67a-7f5aec1939d1/confirm_enrolment
```

Comment:
The application IDs are now 10 chars: https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training-tech-docs/pull/35